source/transactions/tests/retryable-commit-errorLabels.yml

```diff
@@ -0,0 +1,261 @@
+runOn:
+    -
+        minServerVersion: "4.3.0"
+        topology: ["replicaset", "sharded"]
+
+database_name: &database_name "transaction-tests"
+collection_name: &collection_name "test"
+
+data: []
+
+tests:
+  - description: commitTransaction fails with RetryableWriteError label after two network exceptions
+    clientOptions:
+      retryWrites: false
+    failPoint:
+      configureFailPoint: failCommand
+      mode: { times: 2 }
+      data:
+        failCommands: ["commitTransaction"]
+        closeConnection: true
+    operations:
+      - name: startTransaction
+        object: session0
+      - name: insertOne
+        object: collection
+        arguments:
+        session: session0
+        document:
+            _id: 1
+        result:
+        insertedId: 1
+      - name: commitTransaction
+        object: session0
+        result:
+        errorLabelsContain: ["RetryableWriteError", "UnknownTransactionCommitResult"]
+        errorLabelsOmit: ["TransientTransactionError"]
+    expectations:
+      - command_started_event:
+          command:
+            insert: *collection_name
+            documents:
+            - _id: 1
+            ordered: true
+            readConcern:
+            lsid: session0
+            txnNumber:
+            $numberLong: "1"
+            startTransaction: true
+            autocommit: false
+            writeConcern:
+          command_name: insert
+          database_name: *database_name
+      - command_started_event:
+          command:
+            commitTransaction: 1
+            lsid: session0
+            txnNumber:
+            $numberLong: "1"
+            startTransaction:
+            autocommit: false
+            writeConcern:
+          command_name: commitTransaction
+          database_name: admin
+      - command_started_event:
+          command:
+            commitTransaction: 1
+            lsid: session0
+            txnNumber:
+            $numberLong: "1"
+            startTransaction:
+            autocommit: false
+            writeConcern:
+          command_name: commitTransaction
+          database_name: admin
+    outcome:
+        collection:
+            data:
+                - _id: 1
+
+  - description: commitTransaction retries once after a network exception
```

Comment:
Does this add anything of value over the existing "commitTransaction succeeds after connection error" test?

Reply:
No, it does not!